Personal-Team-HealthCheckup/doctorApp
# Enhancement]: Implemented the Dashboard

What does this PR do?
This PR implements the dashboard for the application. The dashboard provides an overview of key metrics and features, improving the overall user experience and navigation within the app.
Changes Made:
Created the dashboard component:
Added a new Dashboard.js file to handle the UI and logic.
Integrated data visualization using charts, lists, and key metrics displays.
Integrated API calls:
Fetched data from the backend to populate the dashboard with dynamic content.
Added routing:
Updated the routing configuration to include the new dashboard page.
UI/UX Enhancements:
Styled the dashboard using Tailwind CSS for responsiveness.
Improved user navigation with clear call-to-action buttons and links to relevant sections.
How to Test:
Steps to test:
Pull the latest changes from this PR.
Run yarn to install any new dependencies.
Navigate to the dashboard by accessing /dashboard route.
Verify that the dashboard displays the correct data, and all links/buttons are functional.
